frontend/src/bundles/auth/pages/auth.tsx

```diff
@@ -59,19 +59,24 @@ const Auth: React.FC = () => {
                     <div className={styles.authWrapper}>
                         <div className={styles.authContent}>
                             <div className={styles.authHeader}>
-                                <h2 className={styles.headerText}>
+                                <h2 className={styles.headerTitle}>
                                     {getText(pathname, 'title')}
                                 </h2>
-                                <Link
-                                    className={styles.headerLink}
-                                    to={
-                                        pathname === AppRoute.SIGN_IN
-                                            ? AuthApiPath.SIGN_UP
-                                            : AuthApiPath.SIGN_IN
-                                    }
-                                >
-                                    {getText(pathname, 'header')}
-                                </Link>
+                                <div className={styles.headerText}>
+                                    <span>
+                                        {getText(pathname, 'headerText')}
+                                    </span>
+                                    <Link
+                                        className={styles.headerLink}
+                                        to={
+                                            pathname === AppRoute.SIGN_IN
+                                                ? AuthApiPath.SIGN_UP
+                                                : AuthApiPath.SIGN_IN
```

Comment:
It's better to create a variable than to write the whole condition in the layout